papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraOAuthTokenFactory.java

```diff
@@ -0,0 +1,59 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthRsaSigner;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.api.client.util.Base64;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+
+@SuppressWarnings({"PMD.BeanMembersShouldSerialize"})
+class JiraOAuthTokenFactory {
+
+    private final URL accessTokenUrl;
+    private final URL temporaryTokenUrl;
+
+    JiraOAuthTokenFactory(URL jiraBaseUrl) throws MalformedURLException {
+        this.accessTokenUrl = new URL(jiraBaseUrl, "/plugins/servlet/oauth/access-token");
+        this.temporaryTokenUrl = new URL(jiraBaseUrl, "/plugins/servlet/oauth/request-token");
+    }
+
+    JiraOAuthGetAccessToken getJiraOAuthGetAccessToken(String jiraVerCode, String tmpToken,
+            String consumerKey, String privateKey) throws NoSuchAlgorithmException, InvalidKeySpecException {
```

Comment:
Tu by się `consumerKey` zmieściło w linijce wyżej. Generalnie w sumie tę stałą można by dać tutaj w ogóle - `"OauthKey"` nie wydaje się być jakiś bardzo śmiałym założeniem w klasie `JiraOAuthTokenFactory`, a IntelliJ agresywnie to podkreśla

Reply:
Przeniesiono do  JiraOAuthTokenFactory. 
Widziałem, że podświetla, ale nie uznałem tego za coś co koniecznie dobrze jest ukrywać w `JiraOAuthTokenFactory` (nawet mimo tego, że to stała i to wykorzystywana tylko tam)